README.md

```diff
@@ -83,6 +83,7 @@ A curated list of all things anime - streaming/downloading/reading/tracking/foru
 * [crunchyroll.com](https://www.crunchyroll.com/)
 * [darkanime.stream](https://darkanime.stream)
 * [dubbedtv.tv](https://www.dubbedtv.tv/)
+* [funimation.com](https://www.funimation.com/)
```

Comment:
Maybe we should add this is paid and restricted to certain countries

Reply:
I didn't write any comments because it is similar to crunchyroll and crunchyroll doesn't have any comments like that.

Reply:
oh good catch, I wasn't aware of cruncyroll, let's add where relevant 💯